src/main/kotlin/racingcar/model/Car.kt

```diff
@@ -0,0 +1,17 @@
+package racingcar.model
+
+import camp.nextstep.edu.missionutils.Randoms.pickNumberInRange
+
+class Car(private val name: String) {
+    private var distance: Int = 0
+
+    fun move() {
+        if (pickNumberInRange(0, 9) >= 4) {
```

Comment:
랜덤 번호에 의존 되어있는 것 같은데 따로 분리를 하여 Boolean 값을 받아오는게 좋을 것 같습니다!

Reply:
0,9,4 를 따로 상수로 분리해서 관리하면 어떨까요??
```suggestion
        private const val MAX_RANDOM_VALUE = 9
        private const val MIN_RANDOM_VALUE = 0
        private const val MIN_MOVE_VALUE = 4

        if (pickNumberInRange(MIN_RANDOM_VALUE, MAX_RANDOM_VALUE) >= MIN_MOVE_VALUE) {
```

Reply:
저도 랜덤 모듈이 바뀔 가능성을 고려하여 랜덤 번호를 추출하는 부분은 분리를 하는 것이 좋다고 생각합니다.